server/src/test/java/org/elasticsearch/index/mapper/DocumentParserTests.java

```diff
@@ -1859,6 +1859,18 @@ public void testBlankFieldNamesSubobjectsFalse() throws Exception {
         }
     }
 
+    public void testBlankFieldNamesSubobjectsFalse() throws Exception {
+        DocumentMapper mapper = createDocumentMapper(topMapping(b -> b.field("subobjects", false)));
+        {
+            MapperParsingException err = expectThrows(MapperParsingException.class, () -> mapper.parse(source(b -> b.field("", "foo"))));
+            assertThat(err.getMessage(), containsString("Field name cannot contain only whitespace: []"));
+        }
+        {
+            MapperParsingException err = expectThrows(MapperParsingException.class, () -> mapper.parse(source(b -> b.field("  ", "foo"))));
+            assertThat(err.getMessage(), containsString("Field name cannot contain only whitespace: [  ]"));
+        }
+    }
+
```

Comment:
Is this additional test still needed given the tests I am adding with #91043 ?

Reply:
No, reverted.